spaces/S000065/properties/P000199.md

```diff
@@ -0,0 +1,9 @@
+---
+space: S000065
+property: P000199
+value: true
+---
+
+Let $I$ denenote the unit interval parameterized by $x$, and let $[0, 1]$ denote the unit interval parameterized by $t$. Let $F : I \times [0, 1] \to I$ denote the homotopy $(x, t) \mapsto (1 - t) x$ from the identity to the constant $0$ map. Let $q : I \sqcup I \to X$ denote the quotient map that defines $X$.
+
+Obtain a homotopy $G : (I \sqcup I) \times [0, 1] \to I \sqcup I$ by composing $F \sqcup F$ with the available canonical map. Since $q \circ G$ is constant on the fibers of $q \times \mathrm{Id}_{[0, 1]}$, it follows that $q \circ G$ descends to a continuous map $X \times [0, 1] \to X$. This is a homotopy from the identity to the constant $0$ map because $F$ is.
```

Comment:
```suggestion
Let $I$ denote the unit interval parameterized by $x$, and let $[0, 1]$ denote the unit interval parameterized by $t$. Let $F : I \times [0, 1] \to I$ denote the homotopy $(x, t) \mapsto (1 - t) x$ from the identity to the constant $0$ map. Let $q : I \sqcup I \to X$ denote the quotient map that defines $X$.

Obtain a homotopy $G : (I \sqcup I) \times [0, 1] \to I \sqcup I$ by composing $F \sqcup F$ with the obvious canonical map. Since $q \circ G$ is constant on the fibers of the quotient map $q \times \mathrm{Id}_{[0, 1]}$, it follows that $q \circ G$ descends to a continuous map $X \times [0, 1] \to X$. This is a homotopy from the identity to the constant $0$ map.
```
Added that $q\times \mathrm{Id}_{[0, 1]}$ is a quotient map, as that's relevant to the argument.  Note to self: It's the usual Whitehead theorem I think, since $[0,1]$ is locally compact Hausdorff (or more simply, it's compact Hausdorff).